appeals/web/src/server/appeals/appeal-details/interested-party-comments/add-ip-comment/add-ip-comment.mapper.js

```diff
@@ -10,6 +10,8 @@ import {
 import { DOCUMENT_STAGE, DOCUMENT_TYPE } from './add-ip-comment.service.js';
 
 /** @typedef {import("../../appeal-details.types.js").WebAppeal} Appeal */
+/** @typedef {import('#appeals/appeal-details/interested-party-comments/interested-party-comments.types').interestedPartyComment} ipComment */
```

Comment:
Types should be upper case
```suggestion
/** @typedef {import('#appeals/appeal-details/interested-party-comments/interested-party-comments.types').interestedPartyComment} IpComment */
```